Desktop/src/routes/Models.svelte

```diff
@@ -68,7 +68,7 @@
         </div>
         {#if selected}
         <!-- svelte-ignore a11y-click-events-have-key-events -->
-          <div class="model-content-overlay rounded-lg" on:click={closeModelContent}>
+          <div class="model-content-overlay rounded-lg" on:click{closeModelContent()}>
```

Comment:
Does this still work?

Just asking since there is no "=" before the "on:click", and would it not still work without the "()".

Reply:
Error. now resolved